packages/fxa-settings/src/components/Settings/Modal/en.ftl

```diff
@@ -2,3 +2,4 @@
 
 modal-close-title = Close
 modal-cancel-button = Cancel
+modal-default-confirm-button = Confirm
```

Comment:
What context is this being used in? Is this confirm as in to confirm your choice, confirm a log in, etc?

Reply:
This is tricky because it's non-specific - this is the default button text for a generic modal that could be used for various purposes. I haven't actually found any instances where the default is used (vs passing in custom text), but thought it would be good to have it translated on the off chance that it does make it's way into a user-facing page. 

I wasn't sure what to add in as a comment that would help localizers here beyond the id itself - I'm absolutely open to suggestions if you have ideas. 🙂

Reply:
@vpomerleau - thanks for adding the comment we discussed in slack. Looks good 👍